index.d.ts

```diff
@@ -41,6 +41,11 @@ export interface FastifyStaticOptions {
   redirect?: boolean;
   wildcard?: boolean | string;
   list?: boolean | ListOptions;
+  /**
+   * @description
+   * Opt-in to looking for pre-compressed files
+   */
+  preCompressed?: boolean;
```

Comment:
This needs a test for the type

Reply:
How can this be accomplished? I see `test/types/index.ts` but unclear about how to write out the test. Is it merely using the new part of the API?

```ts
const options: FastifyStaticOptions = {
  acceptRanges: true,
  cacheControl: true,
  // ...
  preCompressed: false
}
```

Reply:
just do the above